roles/aap_ocp_install/tasks/pre-validate-eda.yml

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
There should be a few more pre-validation tasks such as checking the namespace name and link text if they are defined. Have a look at the checks in the `pre-validate-controller.yml` and use them as a reference to implement them for EDA (not all of them will be applicable to EDA).

Reply:
Added pre-validation checks for namespace, link text and replicas values.